chingsley/ireporter-react
# #165328585 Create and render the components required f…

#### What does this PR do?
It creates and renders all the components required for the app

#### Description of the tasks to be completed?
- create all the required pages
- create and render all the required components

#### How should this be manually tested?
- visit the app at https://chingsley-ireporter-react.herokuapp.com/
- click on the links to see the pages with the components that make up that page

#### What are the relevant pivotal tracker stories?
- https://www.pivotaltracker.com/story/show/165328585
